package/src/integration.ts

```diff
@@ -22,8 +22,14 @@ function getDtsContent(
 		if (!(type === "page" || type === "endpoint")) {
 			continue;
 		}
-		const pattern = `${withoutTrailingSlash(base)}${route.pattern}`;
 		const segments = route.segments.flat();
+		const pattern = `${withoutTrailingSlash(base)}${segments
```

Comment:
Can you add a comment with the source of the original code?

Reply:
Sure thing! Added a short note with the reason for the change too.